src/ml_gym/frontend/dashboard/src/features/jobsStatus/jobsStatusSlice.tsx

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { JobStatusMessageCollectionType, JobStatusType, JobStatusRowType} from '../../app/datatypes'
+import type { RootState } from '../../app/store';
+
+
+
+const initialState = {
+    messages: [],
+    job_id_to_latest_message_index: {}
+} as JobStatusMessageCollectionType
+
+
+const jobsStatusSlice = createSlice({
+    name: 'jobsStatus',
+    initialState,
+    reducers: {
+        jobStatusAdded(state, action) {
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
    jobStatusAdded(state, action) {
```